test/test-test262.js

```diff
@@ -192,6 +170,10 @@ function injectTest262Harness(globalObject, t) {
     throw 'Test262: This statement should not be evaluated.';
   }
 
+  // This cleans up (some of) the global state between tests.
+  // TODO: Have proper isolation.
+  delete globalObject.test262;
+
```

Comment:
We should not fix the test by doing something like that. This masquerade a valid problem with the system. This is not part of the offcial test262 harness.

Reply:
Reverted.